### PR TITLE
[AGENT] GRPC change version from 0 to current timestamp

### PR DESCRIPTION
**Pull request by StrayLittlePunk**
**Phenomenon and reproduction steps**

**Root cause and solution**

GRPC change version from 0 to current timestamp

**Impactions**

**Test method**

**Affected branch(es)**

* main

**Checklist**

- [ ] Dependencies update required
- [ ] Common bug (similar problem in other repo)

### CODE REVIEW

File: agent/src/rpc/synchronizer.rs [hunk 1]
```diff
@@ -146,9 +146,18 @@ impl Default for Status {
...
+                .unwrap()
+                .as_secs(),
+            version_acls: SystemTime::now()
+                .duration_since(SystemTime::UNIX_EPOCH)
+                .unwrap()
+                .as_secs(),
+            version_groups: SystemTime::now()
+                .duration_since(SystemTime::UNIX_EPOCH)
+                .unwrap()
+                .as_secs(),
```
Comment (sharang) [inline agent/src/rpc/synchronizer.rs:160]: 这三个version是接收方version，初始为0没什么问题，可以精确的表达【自己还没有】的含义

File: agent/src/rpc/synchronizer.rs [hunk 2]
```diff
@@ -1089,7 +1098,7 @@ impl Synchronizer {
                     if inner_client.is_none() {
                         session.set_request_failed(true);
                         info!("grpc sync client not connected");
-                        time::sleep(Duration::new(1, 0)).await;
+                        time::sleep(RPC_RETRY_INTERVAL).await;
```
Comment (sharang) [inline agent/src/rpc/synchronizer.rs:1101]: 这个文件的其他的 sleep 1s 也要检查下，也有隐患

File: agent/src/rpc/synchronizer.rs (hunk 1 above)
Comment (StrayLittlePunk) [reply agent/src/rpc/synchronizer.rs]: ok，已更改

File: agent/src/rpc/synchronizer.rs (hunk 2 above)
Comment (StrayLittlePunk) [reply agent/src/rpc/synchronizer.rs:1091]: ok

**Review by sharang — APPROVED** [review]

**Pull request merged**